pkg/apis/certmanager/v1alpha2/types.go

```diff
@@ -98,6 +98,12 @@ const (
 	AllowsInjectionFromSecretAnnotation = "cert-manager.io/allow-direct-injection"
 )
 
+// Issuer specific Annotations
+const (
+	// VenafiCustomFields is the annotation that passes on JSON encoded custom fields to the Venafi issuer
+	VenafiCustomFields = "venafi.cert-manager.io/custom-fields"
```

Comment:
Can you suffix the name of this var with `AnnotationKey` to make `VenafiCustomFieldsAnnotationKey`?

Reply:
Could you also include an example value that this could be set to? We don't actually provide the struct in this package (or exported anywhere), so it's worth at least providing an example value 😄